src/main/resources/assets/firmalife/recipes/cherry_fence.json

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "XBX",
+    "XBX"
+  ],
+  "key": {
+    "X": {
+      "type": "forge:ore_dict",
+      "ore" : "poleCherry"
+    },
+    "B": {
+      "type": "forge:ore_dict",
+      "ore" : "fruitLeaf"
```

Comment:
requiring fruit leaves to build fences is rather expensive.

Reply:
Fruit Poles don't have an intermediary step like logs > planks or planks > plank blocks, getting poles in general is quite expensive right now as a fruit tree can yield a maximum of 5 unless we bump up drop rate.  As it stands fruit wood is a rare and difficult to aquire block group